### PR TITLE
Documented the appendToFile() method

### LINKED ISSUE #7341: [Filesystem] document the new appendToFile() method
see symfony/symfony#20612

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7341.

### CODE REVIEW

File: components/filesystem.rst [hunk 1]
```diff
@@ -283,6 +283,17 @@ complete new file (but never a partially-written file)::
...
 
+appendToFile
+~~~~~~~~~~~~
+
+:method:`Symfony\\Component\\Filesystem\\Filesystem::appendToFile` adds new
+contents at the end of some file::
+
+    $fs->appendToFile('logs.txt', 'Email sent to user@example.com');
+
+If either the file or its containing directory don't exist, this method creates
```
Comment (xabbuh) [inline components/filesystem.rst]: doesn't

**Comment by xabbuh** [comment]
We should also add a `versionadded` directive.

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by chalasr** [comment]
👍  Thanks!

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**